docs/install-gitlab.md

```diff
@@ -22,7 +22,8 @@ frogbot-scan:
       # Creating fix pull requests will be triggered by any push to the default branch.
```

Comment:
Creating fix pull requests will be triggered by any push to the default branch.
-->
Repository scanning is triggered by any push to the default branch.